sv_pipeline/load_data.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python
+
+import argparse
```

Comment:
I think I fixed this, let me know if I made any mistakes

Reply:
ooh, thanks! I ran pylint to check, and it seems to think `from tqdm import tqdm` should be before `from elasticsearch import helpers as es_helpers`. this looks good to me otherwise!